### PR TITLE
Add Promise.any

### PR DESCRIPTION
**Pull request by Elchi3**
For https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/any

### CODE REVIEW

File: live-examples/js-examples/promise/promise-any.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const promise1 = Promise.reject(0);
+const promise2 = new Promise((resolve) => setTimeout(resolve, 100, 'quick'));
+const promise3 = new Promise((resolve) => setTimeout(resolve, 500, 'slow'));
+const promises = [promise1, promise2, promise3];
```
Comment (wbamberg) [inline live-examples/js-examples/promise/promise-any.js:4]: Super-nit: I'd like a blank line between lines 3 and 4, to separate the three promise declarations from the array declaration.

File: live-examples/js-examples/promise/promise-any.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+Promise.any(promises).then((value) => console.log(value));
+
+// expected output:
+// "quick"
```
Comment (wbamberg) [inline live-examples/js-examples/promise/promise-any.js:9]: I think this should be on one line, like:

```
// expected output: "quick"
```

...is there a reason I'm missing why you are not doing this here?

Comment (Elchi3) [reply live-examples/js-examples/promise/promise-any.js]: No reason, just a result of moving things around in the process and not cleaning up afterwards. :)

**Comment by Elchi3** [comment]
Updated, thanks Will! :+1:

**Pull request merged**